bindings/rust/src/bindings/generated.rs

```diff
@@ -84,32 +85,32 @@ pub enum C_KZG_RET {
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub struct FFTSettings {
-    #[doc = "< The maximum size of FFT these settings support, a power of 2."]
+    #[doc = " The maximum size of FFT these settings support, a power of 2."]
     max_width: u64,
-    #[doc = "< Ascending powers of the root of unity, size `width + 1`."]
-    expanded_roots_of_unity: *const fr_t,
-    #[doc = "< Descending powers of the root of unity, size `width + 1`."]
-    reverse_roots_of_unity: *const fr_t,
-    #[doc = "< Powers of the root of unity in bit-reversal permutation, size `width`."]
-    roots_of_unity: *const fr_t,
+    #[doc = " Ascending powers of the root of unity, length `max_width + 1`."]
+    expanded_roots_of_unity: *mut fr_t,
+    #[doc = " Descending powers of the root of unity, length `max_width + 1`."]
+    reverse_roots_of_unity: *mut fr_t,
+    #[doc = " Powers of the root of unity in bit-reversal permutation order, length\n `max_width`."]
+    roots_of_unity: *mut fr_t,
 }
 #[doc = " Stores the setup and parameters needed for computing KZG proofs."]
 #[repr(C)]
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
 pub struct KZGSettings {
-    #[doc = "< The corresponding settings for performing FFTs."]
-    fs: *const FFTSettings,
-    #[doc = "< G1 group elements from the trusted setup, in Lagrange form bit-reversal permutation."]
-    g1_values: *const g1_t,
-    #[doc = "< G2 group elements from the trusted setup; both arrays have FIELD_ELEMENTS_PER_BLOB elements."]
-    g2_values: *const g2_t,
+    #[doc = " The corresponding settings for performing FFTs."]
+    fs: *mut FFTSettings,
+    #[doc = " G1 group elements from the trusted setup,\n in Lagrange form bit-reversal permutation."]
+    g1_values: *mut g1_t,
+    #[doc = " G2 group elements from the trusted setup;\n both arrays have `FIELD_ELEMENTS_PER_BLOB` elements."]
+    g2_values: *mut g2_t,
 }
 extern "C" {
     pub fn load_trusted_setup(
         out: *mut KZGSettings,
-        g1_bytes: *const u8, /* n1 * 48 bytes */
```

Comment:
I guess these were helper comments that will no longer be around. I guess we are fine with this.

Reply:
I think this is due to some change in bindgen that now they do not get generated. I was also surprised by this, but seems an acceptable change to me

Reply:
Yeah this is fine with me too.